website/source/docs/job-specification/group.html.md

```diff
@@ -65,6 +65,13 @@ job "docs" {
   all tasks in this group. If omitted, a default policy exists for each job
   type, which can be found in the [restart stanza documentation][restart].
 
+- `shutdown_delay` `(string: "0s")` - Specifies the duration to wait when
+  killing a groups tasks between removing it from Consul and sending each task
+  a shutdown signal. Ideally services would fail healthchecks once they receive
+  a shutdown signal. Alternatively `shutdown_delay` may be set to give
+  in flight requests time to complete before shutting down. A group level
+  `shutdown_delay` will run regardless if there are any defined group services.
```

Comment:
We should also link to the task shutdown delay and reverse link from there to here.

Here something like:

> In addition tasks may have their own [`shutdown_delay`](/404) which waits between deregistering task services and stopping the task.

And on the task side:

> In addition task groups may have their own [`shutdown_delay`](/404) which waits between deregistering group services and stopping tasks.